src/start/AuthScreen.js

```diff
@@ -284,7 +284,12 @@ class AuthScreenInner extends PureComponent<Props> {
       id_token: credential.identityToken,
     });
 
-    openLinkEmbedded(new URL(`/complete/apple/?${params}`, this.props.realm).toString());
+    openLinkEmbedded(
+      new URL(
+        `/complete/apple/?${params}`,
+        this.props.route.params.serverSettings.realm_uri,
```

Comment:
nit: this is rather a mouthful :slightly_smiling_face: 

Perhaps write something like
```js
  const { serverSettings } = this.props.route.params;
```
here and in some of these other methods, so that these expressions that then go on to do something with the `realm_uri` can remain simpler.

Reply:
Sounds good. For my next revision, I think I'll add intermediate declarations like that as a separate commit, to leave this commit as clear as possible that we're not accidentally involving some other URL besides the intended realm URL, since for privacy it's important to track where credentials go.